bitnami/cassandra/templates/service.yaml

```diff
@@ -23,6 +20,9 @@ spec:
   - name: cql
     port: {{ .Values.service.port }}
     targetPort: cql
+    {{- if .Values.service.nodePort }}
+    nodePort: {{ .Values.service.nodePort }}
```

Comment:
Take into account what we did with WordPress for example:

https://github.com/helm/charts/pull/13009